config/locales/jobs.yml

```diff
@@ -242,6 +242,7 @@ en:
     how_to_apply: Application instructions
 
     job_summary: Job summary
```

Comment:
I guess you can remove job_summary on line 244

Reply:
unfortunately not! We label it as "Job summay" for jobseekers 🤷